### ISSUE TITLE
Documentation: HOWTO install the react component in a plain HTML page

### ISSUE THREAD

**Issue by pacharanero** [rcpch/digital-growth-charts-react-component-library#33]
Based on today's chat with implementers it would be useful to have a page in the documentation which explains how to obtain a single JS file which can be included via `<script src="index.js"></script>` into a normal HTML page and interacted with using plain old JS.

**Comment by dc2007git** [comment]
@raxxius I don't seem to be able to assign you to this issue, but this is related to your implementation of the component in html, so tagging you here just to keep track of the issue!

**Comment by Raxxius** [comment]
Yes this issue is somewhat addressed in the React-client repository under issue 91.

https://github.com/rcpch/digital-growth-charts-react-client/issues/91#issuecomment-1833982687

Can make it generic for a standard html file when I'm next working.

**Comment by eatyourpeas** [comment]
closed #147
